### PR TITLE
📓 Updated HUN

### PR DESCRIPTION
**Pull request by KristofKekesi**

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -1944,21 +1944,13 @@
...
           <target state="needs-review-translation" state-qualifier="tm-suggestion">Mégse</target>
         </trans-unit>
-        <trans-unit id="StatusCenter.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
-          <source>Ongoing Tasks</source>
-          <target state="new">Ongoing Tasks</target>
-        </trans-unit>
-        <trans-unit id="StatusCenterFlyoutTitle.Text" translate="yes" xml:space="preserve">
-          <source>Ongoing Tasks</source>
-          <target state="new">Ongoing Tasks</target>
-        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.hu-HU.xlf:1960]: @KristofKekesi It looks like these strings were accidently removed in your commit, can you revert this change?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.hu-HU.xlf (hunk 1 above)
Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf:1960]: Sorry for that. I didn't compare before pull.

Comment (KristofKekesi) [reply Files/MultilingualResources/Files.hu-HU.xlf:1960]: Fixed

**Review by yaichenbaum — APPROVED** [review]
